dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/MockedBeamSqlTable.java

```diff
@@ -49,6 +48,7 @@
   public static final ConcurrentLinkedQueue<BeamSqlRow> CONTENT = new ConcurrentLinkedQueue<>();
 
   private List<BeamSqlRow> inputRecords;
+  private PCollection.IsBounded isBounded = PCollection.IsBounded.BOUNDED;
```

Comment:
I don't know if you can change the BOUNDED/UNBOUNDED behavior of PCollection by set it here. The test for UNBOUNDED is more about triggers/watermark to me.